.github/workflows/yamllint.yml

```
@@ -0,0 +1,22 @@
+      - "**.yaml"
+      - "**.yml"
+
+jobs:
+  yamllint:
+    name: 🧹 yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - name: ⤵️ Check out configuration from GitHub
+        uses: actions/checkout@v4.2.2
+      - name: 🚀 Run yamllint
+        run: yamllint --strict .
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)